src/host/config/KBoxConfigParser.cpp

```diff
@@ -46,7 +46,15 @@ void KBoxConfigParser::defaultConfig(KBoxConfig &config) {
   config.serial2Config.inputMode = SerialModeNMEA;
   config.serial2Config.outputMode = SerialModeDisabled;
 
-  config.imuConfig.enabled = true;
+  config.imuConfig.enabled = true;            // enable internal IMU sensor
+  config.imuConfig.enabledHdg = true;         // true if values taken from internal sensor
+  config.imuConfig.enabledHeelPitch = true;   // true if values taken from internal sensor
+  config.imuConfig.calHdg = 2;                // hdg valid, if calibration value greater equal
```

Comment:
Do you think anyone will ever want to change this? I am not sure we need this.

Reply:
:-)
E.g.: You had cal 3 for Heading, I took (including) 2 so at least we two have different values.
Same for hdg - I do not want hdg from the BNO055, but heel/pitch will be nice to have.
Especially later, when Leeway is calculated in KBox I will prefer controlled heel-values from KBox